src/windows/views/properties_tableview.py

```diff
@@ -545,6 +534,16 @@ def contextMenuEvent(self, event):
 
             # Add menu options for keyframes
             menu = QMenu(self)
+
+            if self.property_type == "font":
+                font_action = menu.addAction(_("Select font"))
+                font_action.triggered.connect(functools.partial(
+                    self.font_select_callback, cur_property))
+            if self.property_type == "color":
+                color_action = menu.addAction(_("Select color"))
```

Comment:
```suggestion
                color_action = menu.addAction(_("Select a color"))
```
Same deal, this string is already translated.